id/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,8 @@
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
 
+    <uses-permission android:name="org.commcare.dalvik.provider.cases.read" />
```

Comment:
Since access is done in `infra/enrolment-records-store` would it make sense to put the permission it its manifest? This way if we refactor modules there wouldn't be a hidden dependency on other modules.